evaluators/text/BleuEvaluator/__init__.py

```diff
@@ -0,0 +1,38 @@
+from jina.executors.evaluators import BaseEvaluator
+from typing import Sequence, Any
+import nltk.translate.bleu_score as bleu
+import numpy as np
+from nltk import ngrams
+from collections import Counter
+
+
+
+class BleuEvaluator(BaseEvaluator):
+    """
+    :class:`BleuEvaluator`Bilingual Evaluation Understudy Score. 
+    Evaluates the generated sentence (hypothesis) against a reference sentence. 
+    It will use the Bleu on NLTK package.
+    A perfect match will score 1.0 and a perfect unmatch will score 0.0
+    """
+    
+    
+    def get_nltk_bleu_score(self, hypothesis, reference):
+        return bleu.sentence_bleu(reference, hypothesis)
+
+
+    def evaluate(self,
+            hypothesis,
+            reference,
+            nltk_default = True,
```

Comment:
nltk_default is not used. and if it was used it should be a parameter and introduced in the __init__. Learning, be very careful with changing the interface of actual executors methods.

Reply:
Oh yes, I was trying something different and forgot to remove this. Ill take it out